pkg/kubelet/cm/container_manager_stub.go

```diff
@@ -145,7 +145,7 @@ func (cm *containerManagerStub) ShouldResetExtendedResourceCapacity() bool {
 }
 
 func (cm *containerManagerStub) GetAllocateResourcesPodAdmitHandler() lifecycle.PodAdmitHandler {
```

Comment:
Would it make sense to pass ctx here from the caller? It looks like it's available on the caller side.

Reply:
This starts to overlap with https://github.com/kubernetes/kubernetes/pull/132583. I'd like to leave this pull request as tightly aligned to the topologymanager as possible.